pacai/util/probability.py

```diff
@@ -51,23 +55,25 @@ def sample(distribution, values = None):
         distribution = [i[1] for i in items]
         values = [i[0] for i in items]
 
-    if sum(distribution) != 1:
+    if len(distribution) == 0:
+        raise ValueError("Empty distribution")
+
+    if math.isclose(sum(distribution), 1):
         distribution = normalize(distribution)
 
     if values is None:
-        raise ValueError()
+        raise ValueError("Values is None")
```

Comment:
This message needs more context.
Remember, a student will just be getting this and their stack trace.
They would have to look at the code or the docs to understand this message.

We know that we will only get here if they supplied a list, so use that.
Something like: "When sampling from a list, two arguments are required: the probabilities and the values."